example_site/src/components/TemplateBrowser.js

```diff
@@ -185,35 +201,41 @@ export default function TemplateBrowser() {
         <>
           <div className="browser-columns">
             <div className="filters">
-              <h2>Duration</h2>
-              {durationFilters.map((filter) => (
-                <Filter
-                  filter={filter}
-                  filterKey={"duration"}
-                  templateFilter={templateFilter}
-                  setTemplateFilter={setTemplateFilter}
-                />
-              ))}
-
-              <h2>Aspect Ratio</h2>
-              {aspectRatioFilters.map((filter) => (
-                <Filter
-                  filter={filter}
-                  filterKey={"aspectRatio"}
-                  templateFilter={templateFilter}
-                  setTemplateFilter={setTemplateFilter}
-                />
-              ))}
-
-              <h2>Categories</h2>
-              {collections.map((collection) => (
-                <CollectionFilter
-                  collections={collections}
-                  collection={collection}
-                  selectedCollection={selectedCollection}
-                  setSelectedCollection={setSelectedCollection}
-                />
-              ))}
+              <div className="filter-titles">Duration</div>
```

Comment:
Let's change this class name to `filter-title` since it's only styling a single title